public/usage-examples/graphics/has_font/has_font-1-simple.py

```diff
@@ -0,0 +1,19 @@
+from splashkit import *
+
+open_window("Has Font", 800, 600)
+clear_screen()
```

Comment:
This `clear_screen` function produces an error because for python you must pass a colour parameter in the brackets. I tested with `clear_screen(color_white())` and that fixed it